app/views/field_test/experiments/_experiments.html.erb

```diff
@@ -0,0 +1,118 @@
+<style>
+  details details { padding: 1em 0 0 1.5em; }
+</style>
+<% experiments.reverse_each do |experiment| %>
+  <h2><%= experiment.name %><% unless experiment.active? %> <small class="closed">Completed</small><% end %></h2>
+  <details<%= " open" if experiment.active? %>>
+    <summary>Experiment Details</summary>
+
+    <% if experiment.description %>
+      <p class="description"><%= experiment.description %></p>
+    <% end %>
+
+    <details<%= " open" if experiment.active? %>>
+      <summary>Goals for <%= experiment.name %></summary>
+      <% experiment.goals.each do |goal| %>
+        <% results = experiment.results(goal: goal) %>
+
+        <% if experiment.multiple_goals? %>
+          <h3><%= goal.titleize %></h3>
+        <% end %>
+
+        <table>
+          <thead>
+            <tr>
+              <th>Variant</th>
+              <th style="width: 20%;">Participants</th>
+              <th style="width: 20%;">Conversions</th>
+              <th style="width: 20%;">Conversion Rate</th>
+              <th style="width: 20%;">Prob Winning</th>
+            </tr>
+          </thead>
+          <tbody>
+            <% results.each do |variant, result| %>
+              <tr>
+                <td>
+                  <%= variant %>
+                  <% if variant == experiment.winner %>
+                    <span class="check">✓</span>
+                  <% end %>
+                </td>
+                <td><%= result[:participated] %></td>
+                <td><%= result[:converted] %></td>
+                <td>
+                  <% if result[:conversion_rate] %>
+                    <%= (100.0 * result[:conversion_rate]).round(FieldTest.precision) %>%
```

Comment:
This formula might benefit from being a in a helper or something similar as its repeated more than 1 time, but from the description it sounds like you just trying to get a feel for this view hence it may not be blocking.

Reply:
This section was "copied" from the original version.  So I didn't look into it all that much.  But yeah, once we have this in a "Hey product, does this meet the needs, we can re-evaluate"